python/dgl/nn/pytorch/conv/sageconv.py

```diff
@@ -80,8 +93,8 @@ def _lstm_reducer(self, nodes):
         """
         m = nodes.mailbox['m'] # (B, L, D)
         batch_size = m.shape[0]
-        h = (m.new_zeros((1, batch_size, self._in_feats)),
-             m.new_zeros((1, batch_size, self._in_feats)))
+        h = (m.new_zeros((1, batch_size, self._in_src_feats)),
+             m.new_zeros((1, batch_size, self._in_src_feats)))
```

Comment:
_in_dst_feats?

Reply:
The LSTM weights is of shape `(in_src_feats, in_src_feats)`.